### PR TITLE
Feat: Research output amends

### PR DESCRIPTION
**Pull request by kmaid**

### CODE REVIEW

File: apps/frontend/src/library/ResearchOutput.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  if (researchOutputData) {
+    const researchOutput = {
+      ...researchOutputData,
+      team: researchOutputData.team
+        ? {
+            ...researchOutputData.team,
+            href: `/network/teams/${researchOutputData.team.id}`,
+          }
+        : undefined,
+      profileHref: '#',
```
Comment (kmaid) [inline apps/frontend/src/library/ResearchOutput.tsx:25]: Don't know where to link this

Comment (jeysal) [reply apps/frontend/src/library/ResearchOutput.tsx:25]: Yeah that's unfortunate, @fampinheiro shouldn't the research output (response) have some sort of author?

File: apps/frontend/src/library/__tests__/ResearchOutput.test.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+afterEach(() => {
+  nock.cleanAll();
+});
+it('renders the proposal', async () => {
+  nock(API_BASE_URL, {
+    reqheaders: { authorization: 'Bearer token' },
+  })
+    .get('/research-outputs/42')
+    .once()
+    .reply(200, researchOutput);
```
Comment (jeysal) [inline apps/frontend/src/library/__tests__/ResearchOutput.test.tsx:51]: Can I ask you to re-state the `title` here (`{...researchOutput, title: 'asdf'}`) please :)

File: apps/frontend/src/library/__tests__/ResearchOutput.test.tsx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    .reply(200, {
+      ...researchOutput,
+      team: {
+        id: '0d074988-60c3-41e4-9f3a-e40cc65e5f4a',
+        displayName: 'Sulzer, D',
+      },
+    });
+  const { getByText } = await renderComponent('43');
+  expect(getByText('Sulzer, D')).toBeVisible();
+});
```
Comment (jeysal) [inline apps/frontend/src/library/__tests__/ResearchOutput.test.tsx:71]: A test for the generated team href maybe

File: packages/react-components/src/organisms/RichText.tsx [hunk 4]
```diff
@@ -13,17 +13,21 @@ import rehypeToc from 'rehype-toc';
...
+const headline1Spacing = css({ height: `${24 / perRem}em` });
 const components = {
   p: ({ children }: HTMLAttributes<HTMLParagraphElement>) => {
     return <Paragraph>{children}</Paragraph>;
   },
   h1: ({ children, id }: HTMLAttributes<HTMLHeadingElement>) =>
     isTextChildren(children) ? (
-      <Headline2 id={id}>{children}</Headline2>
+      <>
+        <div css={headline1Spacing} />
```
Comment (jeysal) [inline packages/react-components/src/organisms/RichText.tsx:28]: Wrap in a div with padding top instead please

File: packages/react-components/src/templates/ResearchOutputPage.tsx [hunk 5]
```diff
@@ -1,30 +1,124 @@
...
+          <span css={iconStyles}>{chevronCircleLeftIcon}</span> Back
+        </div>
+      </Link>
     </div>
-  );
-};
+    <Card padding={false}>
+      <div css={cardPaddingStyles}>
+        <TagLabel>Proposal</TagLabel>
+        <Headline3 styleAsHeading={1}>{title}</Headline3>
```
Comment (jeysal) [inline packages/react-components/src/templates/ResearchOutputPage.tsx:97]: Why was this changed, `h1` seems correct in the level hierarchy to me?

**Review by jeysal — CHANGES_REQUESTED** [review]
Great work, just a couple of comments

File: packages/react-components/src/templates/ResearchOutputPage.tsx (hunk 5 above)
Comment (kmaid) [reply packages/react-components/src/templates/ResearchOutputPage.tsx:97]: James requested it on the trello card

Comment (jeysal) [reply packages/react-components/src/templates/ResearchOutputPage.tsx:97]: Then you probably meant to write `Display styleAsHeading={3}`

Comment (kmaid) [reply packages/react-components/src/templates/ResearchOutputPage.tsx:97]: Derp.

File: apps/frontend/src/library/ResearchOutput.tsx (hunk 1 above)
Comment (jeysal) [reply apps/frontend/src/library/ResearchOutput.tsx:25]: Oh this effectively appears to be the `ASAP Admin` thing?

Comment (jeysal) [reply apps/frontend/src/library/ResearchOutput.tsx:25]: Do we know where that should go @fampinheiro ?

File: packages/react-components/src/templates/__tests__/ResearchOutputPage.test.tsx [hunk 6]
```diff
@@ -1,19 +1,42 @@
...
+      {...props}
       type="proposal"
-      created="2020-06-25T15:00:47.920Z"
+      title="title"
       text="content"
     />,
   );
   expect(getByText(/proposal/i)).toBeVisible();
-  expect(getByText(/title/i, { selector: 'h1' })).toBeVisible();
+  expect(getByText(/title/i, { selector: 'h3' })).toBeVisible();
```
Comment (jeysal) [inline packages/react-components/src/templates/__tests__/ResearchOutputPage.test.tsx:24]: This shouldn't pass? If this still passes with `h3` then the text `title` you used is probably too generic because it clearly selects something else from the generated text

**Review by jeysal — APPROVED** [review]
LGTM except for the thing that probably doesn't work

File: packages/react-components/src/templates/__tests__/ResearchOutputPage.test.tsx (hunk 6 above)
Comment (kmaid) [reply packages/react-components/src/templates/__tests__/ResearchOutputPage.test.tsx]: It didn't pass 😅

File: apps/frontend/src/library/ResearchOutput.tsx (hunk 1 above)
Comment (fampinheiro) [reply apps/frontend/src/library/ResearchOutput.tsx:25]: The author is a left-over from ROMP, on this case it should link to a section on the website about ASAP.
Can be a simple page like `terms-and-conditions`, but for now will point nowhere :(

One more pending link to add to the list.

Comment (kmaid) [reply apps/frontend/src/library/ResearchOutput.tsx:25]: We do not. Will put into a backlog ticket for when we do
